### PR TITLE
[Doc Improvement][Support deep links to chat messages][2529296]

### PR DESCRIPTION
**Pull request by v-ypalikila**
Added content for deeplinks to chat messages in Teams.

### CODE REVIEW

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 1]
```diff
@@ -337,6 +337,20 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to navigate to a message in individual and group chats in Teams:
+
+`http://teams.microsoft.com/l/message/{chatId}/{messageId}?context={contextObject}`
+
+Example: `http://dev.local/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context={"contextType"%3A"chat"}`
```
Comment (juna-gupta) [inline msteams-platform/concepts/build-and-test/deep-links.md:346]: teams.microsoft.com

Comment (juna-gupta) [reply msteams-platform/concepts/build-and-test/deep-links.md:346]: and confirm if the QP should be URL encoded.

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 2]
```diff
@@ -337,6 +337,20 @@ The query parameters are:
...
+
+The query parameters are:
+
+* `chatId`: ChatId that is available from the tab context for group and meeting chat. For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.
```
Comment (juna-gupta) [inline msteams-platform/concepts/build-and-test/deep-links.md:350]: We should say that apps can read a group chat id through context (JS SDK) or from the incoming request payload to Bot.

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 3]
```diff
@@ -337,6 +337,20 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to navigate to a message in individual and group chats in Teams:
```
Comment (juna-gupta) [inline msteams-platform/concepts/build-and-test/deep-links.md:342]: Do you mean - a 1:1 chat with bot/users?

I'm not sure if we use the term "individual", double check.

**Review by juna-gupta — CHANGES_REQUESTED** [review]
Please address the comments.

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 3 above)
Comment (jofri-msft) [reply msteams-platform/concepts/build-and-test/deep-links.md:342]: can we just say to a message in chat in Teams?

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 1 above)
Comment (jofri-msft) [reply msteams-platform/concepts/build-and-test/deep-links.md:346]: The url should be of this format
?context={"contextType":"chat"}
That whole thing percent encoded is 
?context=%7B%22contextType%22:%22chat%22%7D

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 3 above)
Comment (jofri-msft) [reply msteams-platform/concepts/build-and-test/deep-links.md:342]: can we just say to a message in chat in Teams?

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 2 above)
Comment (jofri-msft) [inline msteams-platform/concepts/build-and-test/deep-links.md:350]: I don't actually see where the chat ID is present in a tab. The way I get it is by going to the web version of teams and taking from the url for that chat. The only other option that I am aware of is to use graph to make a call to list chats

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75e9752564dd589bf774b5a5ad81273817c9bb31/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 93 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4a4cff83-48ac-4146-afc2-166bcef56814) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 3 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/deep-links.md]: Updated individual chat to personal chat.

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 2 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/deep-links.md:350]: True, chat ID can be retrieved from the URL of the chat in Teams web client or using the [Graph API ](https://docs.microsoft.com/en-us/graph/api/chat-get?view=graph-rest-1.0&tabs=http). Updated the same in the document.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/53e1b8b51620bf4029db55d9f571f5d347498b96/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 94 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0340a04c-baf9-4d1a-8153-e158b785da4a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fc5814cdd9aec274592d31af8f5b8912e4bd4bdf/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 94 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1025d85-65c3-4fe1-886a-6b746d741a4f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by jofri-msft — DISMISSED** [review]

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 4]
```diff
@@ -336,6 +336,23 @@ The query parameters are:
...
+
+`http://teams.microsoft.com/l/message/{chatId}/{messageId}?context={"contextType":"chat"}`
+
+Example: `http://teams.microsoft.com/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context=%7B%22contextType%22:%22chat%22%7D`
+
+The query parameters are:
+
+* `chatId`: ChatId of the conversation. You can get the `chatId` from URL of the chat in Teams web client or using the [Microsoft Graph API](/graph/api/chat-get?view=graph-rest-1.0&tabs=http&preserve-view=true). For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.
+
+  Apps can read a group chat ID through context using the Microsoft Teams JavaScript library or from the incoming request payload to Bot.
```
Comment (juna-gupta) [inline msteams-platform/concepts/build-and-test/deep-links.md:351]: Did we validate this? (that the 19:*** chat id is available in tab context / bot payload)

**Review by juna-gupta — APPROVED** [review]
Verification comment.

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 5]
```diff
@@ -336,6 +336,23 @@ The query parameters are:
...
+
+* `messageId`: Message ID of the conversation. For example, `1563480968434`.
+* `contextObject`: Specify the contextType as chat.
```
Comment (aosolis) [inline msteams-platform/concepts/build-and-test/deep-links.md:354]: The query parameter name is "context", not "contextObject"

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 6]
```diff
@@ -336,6 +336,23 @@ The query parameters are:
...
+
+Example: `http://teams.microsoft.com/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context=%7B%22contextType%22:%22chat%22%7D`
+
+The query parameters are:
+
+* `chatId`: ChatId of the conversation. You can get the `chatId` from URL of the chat in Teams web client or using the [Microsoft Graph API](/graph/api/chat-get?view=graph-rest-1.0&tabs=http&preserve-view=true). For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.
+
+  Apps can read a group chat ID through context using the Microsoft Teams JavaScript library or from the incoming request payload to Bot.
+
+* `messageId`: Message ID of the conversation. For example, `1563480968434`.
```
Comment (aosolis) [inline msteams-platform/concepts/build-and-test/deep-links.md:353]: Where does one get this value?

Comment (aosolis) [reply msteams-platform/concepts/build-and-test/deep-links.md:353]: perhaps "ID of the specific message in the conversation"? The conversation itself doesn't have a message ID.

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 5 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/deep-links.md:354]: updated the parameter.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7b67edc91b919a6422b3bf21ad0994de8f8db530/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 92 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/afa59500-b682-4532-b659-3fc536c7d7bb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/662548472d17694c5d651c81f3af5223d2aade49/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 92 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/376b8303-43e1-48d8-9b80-d4223c868201) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/68cfe82af78dbb4cebeadef702174de40c5e9938/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 92 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62ca9ff5-938c-4bde-8d5c-0fa750cf29b4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/464310862129787a58e67a3165a847fb4af67aec/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 92 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c8a8ea73-b7af-4e64-b404-9b9f3b362a75) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 4 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/deep-links.md]: Validated the information with the PM and updated the information.

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 6 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/deep-links.md]: Updated that Each message in a chat has a unique ID. When a bot posts a message in chat, the `messageid` is returned.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/58a8a1f09a3af16ae20391762044b1b5ca275543/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 92 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/328406cd-81a7-4761-bdd7-bd9bd2aeeb3b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 7]
```diff
@@ -336,6 +336,26 @@ The query parameters are:
...
+Example: `http://teams.microsoft.com/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context=%7B%22contextType%22:%22chat%22%7D`
+
+The query parameters are:
+
+* `chatId`: ChatId of the conversation. You can get the `chatId` from URL of the chat in Teams web client or using the [Microsoft Graph APIs](/graph/api/chat-get?view=graph-rest-1.0&tabs=http&preserve-view=true). The supported format for `chatId` is 19:xxx. For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.
+
+  > [!NOTE]
+  > For One-on-One chats with bot, the incoming payload to bot contains the conversation ID in a:xxx format.
+
+  Apps can read a chat ID through app context in Teams, incoming payload to bot, or through Graph APIs.
```
Comment (juna-gupta) [inline msteams-platform/concepts/build-and-test/deep-links.md:354]: Lets put this before the note.

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 8]
```diff
@@ -336,6 +336,26 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to go to a message in personal and group chats in Teams:
```
Comment (juna-gupta) [inline msteams-platform/concepts/build-and-test/deep-links.md:341]: Use this deep link format to navigate a user to a message in a personal or group chat in Teams.

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 9]
```diff
@@ -336,6 +336,26 @@ The query parameters are:
...
+
+`http://teams.microsoft.com/l/message/{chatId}/{messageId}?context={"contextType":"chat"}`
+
+Example: `http://teams.microsoft.com/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context=%7B%22contextType%22:%22chat%22%7D`
+
+The query parameters are:
+
+* `chatId`: ChatId of the conversation. You can get the `chatId` from URL of the chat in Teams web client or using the [Microsoft Graph APIs](/graph/api/chat-get?view=graph-rest-1.0&tabs=http&preserve-view=true). The supported format for `chatId` is 19:xxx. For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.
```
Comment (juna-gupta) [inline msteams-platform/concepts/build-and-test/deep-links.md:349]: Lets fix the order:
ChatId of the conversation. The supported format for `chatId` is 19:xxx. For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.

Apps can read chatId through ***

Note: **

**Review by juna-gupta — CHANGES_REQUESTED** [review]
Please address the comments.

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 7 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/deep-links.md:354]: updated

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 8 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/deep-links.md:341]: updated

File: msteams-platform/concepts/build-and-test/deep-links.md (hunk 9 above)
Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/deep-links.md:349]: Updated the order.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4fcbd4b0305aa38c47e26b90f3a3c14936679978/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 92 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a99b097-94a5-4d3c-83bf-dbf69cb5a8d4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by juna-gupta — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/387440e9379559ff305c3468665a95d3b2dbdf86/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 93 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d22f4868-c6bb-4f52-87e8-6abca80ee21a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fca4f347104918518c2bfff9d8d797ee6b8db491/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/32aeec99-be45-4771-befa-9fe5da3854ed) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5e98e659e8b3800cfc1085e90ae6ee41622a4e73/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a5b74cac-baa7-49a5-962a-6220f17e0f6d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e8622fc0cee3426ebd7e0d633b95cabb6de9dcd1/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 95 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/737b4fc7-36b9-412b-a5a5-e056b0fd693d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2de86853dd5d5739047ccee13807937c41bdd6de/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 95 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2851bf4e-53e4-4607-a6ac-614c4130d87f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by afrahman-MS — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/925137f981d7fc702fa44a01d28b3c2269721878/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 95 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c44d4657-ba2c-46f7-adfa-85e589accf0e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**